### PR TITLE
Create logic module

### PR DESCRIPTION
**Pull request by legafise**

### CODE REVIEW

File: store-logic/pom.xml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>Stores</artifactId>
+        <groupId>com.lashkevich.stores</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
```
Comment (IvanYarochkin) [inline store-logic/pom.xml:11]: Move it below the `artifactId, versopm and packaging`

File: store-logic/pom.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    <artifactId>store-logic</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
```
Comment (IvanYarochkin) [inline store-logic/pom.xml:15]: Use pom packaging

File: store-logic/pom.xml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>mysql</groupId>
+            <artifactId>mysql-connector-java</artifactId>
+        </dependency>
+    </dependencies>
+
```
Comment (IvanYarochkin) [inline store-logic/pom.xml:33]: Remove the space

**Review by IvanYarochkin — CHANGES_REQUESTED** [review]

File: store-logic/pom.xml (hunk 3 above)
Comment (legafise) [reply store-logic/pom.xml:33]: Done

File: store-logic/pom.xml (hunk 2 above)
Comment (legafise) [reply store-logic/pom.xml:15]: Done

File: store-logic/pom.xml (hunk 1 above)
Comment (IvanYarochkin) [reply store-logic/pom.xml:15]: Don't forget to type done or something like this XD

File: store-logic/pom.xml (hunk 2 above)
Comment (IvanYarochkin) [reply store-logic/pom.xml]: The next time type something like: "Keep the packaging jas because ..."

Comment (IvanYarochkin) [reply store-logic/pom.xml]: Or as discussed with Ivan jar packaging is kept

**Review by IvanYarochkin — APPROVED** [review]

**Pull request merged**